src/main/java/com/github/fishio/IEatable.java

```diff
@@ -0,0 +1,37 @@
+package com.github.fishio;
+
+/**
+ * Interface for objects that can eat or can get eaten.
+ */
+public interface IEatable {
+
+	/**
+	 * checks of this object can be eaten by the other object.
+	 * @param other
+	 * 		The object to check for
+	 * @return
+	 * 		True is this can be eaten by other, false otherwise
+	 */
+	boolean canBeEatenBy(IEatable other);
+	
+	/**
+	 * Method called when this is eaten.
+	 */
+	void eat();
```

Comment:
I Imagine that most Eatables will have the same implementation as in PlayerFish and EnemyFish. In fact, because it's the same, wouldn't it be easier to make that a default implementation? If we later have an Eatable that does not follow the size larger than the other rule, that class can simply override it.

<organisation